### PR TITLE
todo attempt Andrew

### PR DESCRIPTION
**Pull request by Darren-Corbett**

### CODE REVIEW

File: index.ios.js [hunk 1]
```diff
@@ -1,30 +1,33 @@
...
-        </Text>
-        <Text style={styles.instructions}>
-          To get started, edit index.ios.js
-        </Text>
-        <Text style={styles.instructions}>
-          Press Cmd+R to reload,{'\n'}
-          Cmd+D or shake for dev menu
-        </Text>
+        <TextInput
+          style={styles.addTodoInput}
```
Comment (Darren-Corbett) [inline index.ios.js:26]: improper use of bind should just
```js
onSumbitEditing={(text) => this.setState({text})}
```

File: index.ios.js [hunk 2]
```diff
@@ -1,30 +1,33 @@
...
 import {
   AppRegistry,
   StyleSheet,
   Text,
-  View
+  View,
+  TextInput,
+  TouchableHighlight
 } from 'react-native';
+import TodoList from './src/components/todoList'
```
Comment (moogus) [inline index.ios.js]: Created a class where a stateless component functions would be fine (best practice)

File: index.ios.js [hunk 3]
```diff
@@ -1,30 +1,33 @@
...
```
Comment (Darren-Corbett) [inline index.ios.js:10]: Created a class where a stateless component functions would be fine (best practice)

File: src/components/todoList.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        {item.text}
+        </Text>
+      );
+    });
+
+    return (
+      <View>{todoList}</View>
+    )
+  }
+}
```
Comment (moogus) [inline src/components/todoList.js]: doesn't understand flexbox

**Pull request closed without merging**